src/SFA.DAS.EmployerAccounts.Web.UnitTests/Controllers/TransfersControllerTests/WhenIViewTheTransferRequestsComponent.cs

```diff
@@ -41,7 +41,7 @@ public void Arrange()
             _mediator = new Mock<IMediator>();
             _mediator.Setup(m => m.SendAsync(_query)).ReturnsAsync(_response);
 
-            _controller = new TransfersController(_logger.Object, _mapper, _mediator.Object);
+            _controller = new TransfersController(_logger.Object, _mapper, _mediator.Object,null);
```

Comment:
spacing

Reply:
No longer an issue as transfersController doesn't need the extra parameter